<!DOCTYPE HTML>
<html lang="en">
  <head>
    <meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="Description" content="The Rafael Casellas Lab combines molecular biology, gene targeting, and transgenic techniques, as well as genomic and bioinformatic tools to unravel molecular mechanisms driving early development and peripheral activation of B lymphocytes"> 
    <meta name="keywords" content="Rafael,Casellas,Regulome,genomics,B cells,NIAMS">

    <title>Casellas Lab Site - NIAMS - Research</title>

        <link rel="stylesheet" href="/assets/css/bootstrap.css" />
        <link rel="stylesheet" href="/assets/css/justified-nav.css" />
        <link rel="stylesheet" href="/assets/css/font-awesome.css" />
	<style>
	.container {
		font-family:"Palatino Linotype";
		font-size:18px;
	}
	.col-lg-12 {
		vertical-align:middle;
		font-family:"Adobe Garamond Pro";
    }
	.thumbnail {
		border: 0 none;
    	box-shadow: none;
	}
	.caption {
		margin-left:10%;margin-right:10%;
	}
	</style>
<!--[if lt IE 9]>
      <script src="/assets/js/html5shiv.js"></script>
      <script src="/assets/js/respond.js"></script>  
<![endif]-->   

</head>
<!--  <body style="background-color: rgb(108,123,139);"> -->
  <body style="background-color: rgb(255,255,255);">
  <div class="container">
	<!-- Navigation Bar and header
	===================================================  -->
      <div class="masthead" style="padding-bottom:0px;">
        <ul class="nav nav-justified">
          <li><a href="/index.html">Home</a></li>
          <li class="active"><a href="/Research/index.html">Research</a></li>
          <li><a href="/People/index.html">People</a></li>
          <li><a href="/Publications/index.html">Publications</a></li>
          <li><a href="/Regulome/index.html">Regulome</a></li>
          <li style="white-space: nowrap"><a href="/GenomeEditing/index.html">Genome Editing</a></li> 
		  <li style="white-space: nowrap"><a href="/News/index.html">News</a>
		  <li><a href="/Join/index.html">Join</a></li>
        </ul>
      </div> <!-- /masthead -->

	<!-- Slideshow
	===================================================  -->
	<div class="container" style="background-color:white;font-family:'Adobe Garamond Pro';font-size:18px;"> 
<br><br>
<p class="lead">The human genome contains ~22,000 genes, which is a modest twofold increase in gene number compared to Drosophila. This finite number of genes poses a challenge for the immune system, which must recognize a nearly infinite variety of immunogens by means of antigen receptor genes. This problem was first recognized by Karl Landsteiner at the Rockefeller University, who discovered the immune system’s capacity to bind essentially any chemical substance, whether natural or synthetic. To achieve unlimited antigen recognition, vertebrates have evolved a series of genetic alterations that target B and T cell receptor genes. In developing B cells for instance, heavy and light chain antibody genes are first assembled from variable (V), diversity (D), and joining (J) gene segments in a site-specific recombination process called V(D)J recombination. This reaction is catalyzed by the RAG1 and RAG2 recombinase complex at specific developmental stages.</p>
<br>

<div class="row" style="padding-left:40px;padding-right:20px;">
	<div class="col-md-6 col-sm-6">
		<div class="thumbnail">
			<img src="/images/slideimages/JEM_2007.png" style="height:400px;width:400px;" class="img-responsive" alt="Rafael Casellas NIAMS Germinal Center">
				<div class="caption"><b>Figure 1. </b>Left: germinal centers at intestinal Peyer’s patches. Activated B cells in this microenvironment express AID (labeled with GFP) which promotes recombination and hypermutation of immunoglobulin genes. Right: plasma cells (labeled with YFP) at mouse intestinal villi.
				</div>
			</img>
		</div>
	</div>
	<div class="col-md-6 col-sm-6">
		<div class="thumbnail">
			<img src="/images/slideimages/7921_13782_Casellas_1496.jpg" style="height:400px;width:400px;" class="img-responsive" alt="Rafael Casellas NIAMS Intestinal villus">
				<div class="caption"><b>Figure 2. </b>Close up view of an intestinal villus showing plasma cells (green), which release milligrams of IgA antibodies per day.
				</div>
			</img>
		</div>
	</div>
</div> <!-- end row -->
<br>
<p class="lead">
Upon migration to the periphery, newly generated B cells may encounter and recognize antigens. This interaction prompt naïve, G0 B cells to enter the cell cycle and undergo a marked increase in cell size, RNA synthesis, and protein output. Within a few days, some of the activated B lymphocytes develop into short-lived plasma cells and secrete low affinity IgM antibodies to counteract the immunogen. Others migrate to follicular areas where they participate in the germinal center reaction (Figure 1). In this specialized microenvironment, immunoglobulin genes undergo class switch recombination and somatic hypermutation. Switch recombination replaces the constant domain of the immunoglobulin heavy chain genes from Cm to Cg, Ce, or Ca, leading to a switch in isotype class from IgM to IgG, IgE, or IgA, respectively. Switching allows antigens to be eliminated by different routes. In the gut for instance, plasma cells release milligrams of IgA antibodies to impede commensal bacteria from crossing the intestinal lumen (Figure 2). The process of hypermutation introduces random single point mutations at immunoglobulin variable domains, and in conjunction with selection, it improves the antibody’s affinity for the antigen in germinal centers. While mechanistically different, switch recombination and hypermutation are initiated by the activation-induced cytidine deaminase (AID) enzyme.
</p>
<p class="lead">
AID promotes recombination and hypermutation by deaminating deoxy-cytidines into uracils. In non-B cells, spontaneous DNA deamination is recognized and faithfully repaired by mismatch repair and base excision repair enzymes. In B cells however, deamination downstream of AID is only partially repaired in an error-free manner. Instead, a significant fraction of U:G mismatches at immunoglobulin switch recombination domains are processed by error-prone mechanisms into mutations or DNA breaks.
</p>
<br>
<div class="row" style="padding-left:40px;padding-right:20px;">
	 <div class="col-md-8 col-sm-8 col-md-offset-2 col-sm-offset-2">
		<div class="thumbnail">
			<img src="/images/Figure3.png" class="img-reponsive" class="img-responsive" alt="Rafael Casellas NIAMS AID targeting">
				<div class="caption" style="margin:0;"><b>Figure 3. </b>AID on-targeting (Ig (immunoglobulin) loci) and off-targeting (non-Ig loci) activity leads to the formation of DNA lesions, which are intermediate to physiological switch recombination or, to a lower extent, cancer forming chromosomal translocations. Burkitt lymphomas are characterized by translocations that juxtapose the Myc oncogene to potent Ig enhancers.
				</div>
			</img>
		</div>
	</div>
</div> <!-- end row -->
<br>
<p class="lead">
While AID preferentially targets Ig genes, other genomic sites, including the Myc and Pim1 oncogenes, undergo low levels of AID-mediated damage. This off-targeting activity can generate cancer-causing translocations by juxtaposing oncogenes to potent immunoglobulin enhancers. For instance, human Burkitt lymphomas and mouse plasmacytomas originate from cells carrying Igh-Myc translocations (Figure 3). This phenomenon provides a rationale to the observation that nearly all human lymphomas diagnosed in the western world are of B cell origin.
</p>
<p class="lead">
Our laboratory is currently working on two important questions concerning the nature of AID targeting: i) why is AID-mediated deamination preferentially recruited to immunoglobulin genes? And ii) why are some oncogenes also susceptible to such an activity? To address these issues our laboratory is taking a genomic approach by exploring the various mechanisms involved in AID targeting: regulation of transcription, DNA damage and repair, and nuclear architecture among others.
</p>

</div>
</div> <!-- container -->  
<br>
</div>
</div> <!-- end container -->
<br><br>        
</div>

<!-- Google Analytics -->
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-47028203-1', 'casellaslab.github.io');
  ga('send', 'pageview');

</script>


                <!--[if lt IE 9]>
                        <script src="/assets/js/require.js"></script>
                <![endif]-->
        <script src="/assets/js/jquery.js"></script>
        <script src="/assets/js/bootstrap.js"></script>
        <!--[if lt IE 9]>
                     <script src="/assets/js/css3-mediaqueries.js"></script>
                <![endif]-->

  </body>
</html>
